hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationManager.java

```diff
@@ -734,6 +804,10 @@ public void migrate(Address source, int sourceCurrentReplicaIndex, int sourceNew
         }
     }
 
+    /**
+     * Invoked on the master node to assert if the partition table is missing some replicas, has more than the maximum
```

Comment:
Actually the comment should be `missing some replicas` as in - they should be non-null if there are no nodes currently shutting down. Expanded the javadoc to explain this.